app/views/static/transcribe_help.html.slim

```diff
@@ -1,7 +1,8 @@
 =render({ :partial => '/shared/work_tabs', :locals => { :selected => 9, :work_id => @work.id }})
 
 .columns
-  article.maincol
+  -direction = Rtl.rtl?(@collection.text_language) ? 'rtl' : 'ltr'
```

Comment:
Transcription help needs to be un-conditionalized, as the project plans to list its instructions in English.